### DISCUSSION TITLE
Certificate of completion

### DISCUSSION THREAD

**Discussion by emoloic** [mrdbourke/pytorch-deep-learning#153] [General]
Hello !!!
I wanted to know whether  is there a chance to get a certificate after watching the video on YouTube.
Thanks.

**Comment by mrdbourke** [comment]
Hi @emoloic,

There is no certificate of completion for the YouTube video.

Other than showcasing what you've done through a GitHub repo or various blog posts sharing what you've learned.

See more here: https://github.com/mrdbourke/pytorch-deep-learning/discussions/154#discussioncomment-4102158
